recipes/pyrad/meta.yaml

```diff
@@ -1,14 +1,18 @@
+{% set name = "pyrad" %}
+{% set version = "3.0.66" %}
+{% set hash = "7dbd67e532058f7b7de76d14cf631fd3e3c841cd80fac4e55fbce8bb52ac6537" %}
+
 package:
-  name: pyrad
-  version: '3.0.64'
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  fn: 3.0.64.tar.gz
-  url: https://github.com/dereneaton/pyrad/archive/3.0.64.tar.gz
-  sha256: 45fd47819489df150427d5e28b8296b7aebae2281b8560412977668475c65796
+  fn: {{ version }}.tar.gz
+  url: https://github.com/dereneaton/pyrad/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
 
 build:
-  skip: True # [not py27]
+  skip: True  # [not py27]
 
 requirements:
   build:
```

Comment:
`build` requirements can be condensed to
```yaml
  build:
    - python
    - setuptools
```
. Also, I don't think six is needed as a `run` requirement.